hazelcast/src/main/java/com/hazelcast/internal/partition/PartitionServiceProxy.java

```diff
@@ -229,8 +229,10 @@ public Member getOwner() {
         @Override
         public int compareTo(Object o) {
             PartitionProxy partition = (PartitionProxy) o;
-            Integer id = partitionId;
-            return (id.compareTo(partition.getPartitionId()));
```

Comment:
This is reported as an issue by spotbugs when target is 1.8, suggesting to use `Integer.compare(int, int)` to avoid boxing, however this method is only available since JDK7. It is replaced with direct integer comparison to avoid boxing.